lib/settings/gunicorn.py

```diff
@@ -0,0 +1,17 @@
+import uptrace
+from opentelemetry.instrumentation.fastapi import FastAPIInstrumentor
+from lib.secrets import Secrets
+
+
+def post_fork(server, worker):  # pylint: disable=unused-argument
+    uptrace.configure_opentelemetry(
+        dsn=Secrets.get_secret("UPTRACE_DSN"),
+        service_name="infinity-api",
+        service_version="1.2.0",
```

Comment:
There's a lot of places we're writing the API version, can we automate it somehow?

Reply:
Good point, I'll think in a way to have a single source of truth for that